openedx_learning/apps/authoring/components/api.py

```diff
@@ -422,7 +420,7 @@ def create_component_version_content(
     content_id: int,
     /,
     key: str,
-    learner_downloadable: bool = False,
+    learner_downloadable: bool = None,
```

Comment:
Okay, change in plan: Let's just kill the argument entirely, and then bump the version to 0.7.0.

Reply:
Sure